src/topology/continuous_function/polynomial.lean

```diff
@@ -4,7 +4,9 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison
 -/
 import topology.algebra.polynomial
-import topology.continuous_function.basic
+import topology.continuous_function.algebra
+import topology.continuous_function.compact
+import topology.unit_interval
 
 /-!
 # Constructions relating polynomial functions and continuous functions.
```

Comment:
The file is a little bit less of a stub now. It won't be able to grow too much, because to state the general Stone-Weierstrass you will need to import another file which imports this one. So it may be time to have a proper file-level docstring now.

Reply:
Good point!